### ISSUE TITLE
Create PyLint Runner that ignores auto-generated code

### ISSUE THREAD

**Issue by WilliamBerryiii** [Azure/azure-iot-cli-extension#6]
Can we add a step in the build process to auto-run pylint (with a custom rule set) across all non-auto-generated files?

**Comment by digimaun** [comment]
Yes, thanks for the suggestion...the CI pipeline is in work which will include pylint and/or flake8 checks.
